packages/database/src/core/util/libs/parser.ts

```diff
@@ -146,6 +146,19 @@ export const parseURL = function(
     } else if (parts[0].slice(0, colonInd).toLowerCase() === 'localhost') {
       domain = 'localhost';
     }
+    // Support `ns` query param if subdomain not already set
```

Comment:
Can we add a `queryString` similar to the `pathString` above?

Reply:
done